From b81c96aa9569be8a2516cfd7e064c51a8da02e66 Mon Sep 17 00:00:00 2001
From: Moritz 'e1mo' Fromm <git@e1mo.de>
Date: Mon, 2 Jan 2023 18:27:36 +0100
Subject: [PATCH] Fix settings page error due to unwritable .env file

---
 app/Misc/Helper.php | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/app/Misc/Helper.php b/app/Misc/Helper.php
index 1abb62f3..57370cb5 100644
--- a/app/Misc/Helper.php
+++ b/app/Misc/Helper.php
@@ -917,7 +917,13 @@ class Helper
                 $contents = $contents."\n{$key}={$value}\n";
             }
         }
-        \File::put($env_path, $contents);
+        #\Session::flash('flash_error_unescaped', "Unable to write $env_path file for key $key with content of <pre>$value</pre>");
+        try {
+          \File::put($env_path, $contents);
+        } catch (\Exception $e) {
+          \Helper::logException($e, 'Error updating ' . $key  .' in .env file: ');
+          \Session::flash('flash_error_unescaped', "Unable to write settings to <code>$env_path</code>: " . $e->getMessage());
+        }
     }

     /**
--
2.38.1
